import.sh

```diff
@@ -108,7 +108,7 @@ pb() {
     else
         pecho "using debugging version"
         if ! [ -e ~/.papersilent ]; then
-            cat ~/workspace/bash/"$PAPERPACKAGE" || (echo "debug package not found" && return 1)
+            cat ~/workspace/bash/"$PAPERPACKAGE" || { echo "debug package not found" && return 1; }
```

Comment:
You were creating a whole new subshell, but I think all you wanted to do was group the commands together, so I've done that here, and further down.